liota/dccs/iotcc.py

```diff
@@ -0,0 +1,248 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yes, the naming mechanism has to be decided together.IoTControlEnterprise makes sense then ProjectIce in the liota code base.